src/main/java/Parser.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Parser takes in a user command, parses it and returns the corresponding command.
+ */
+public class Parser {
+
+    private TaskList taskList;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructor method
+     * @param taskList list of tasks
+     * @param ui standard ui object
+     * @param storage standard storage object
+     */
+    public Parser(TaskList taskList, Ui ui, Storage storage) {
+        this.taskList = taskList;
+        this.ui = ui;
+        this.storage = storage;
+    }
+
+    /**
+     * Parse method takes in command in the form of a string and returns a class.
+     * @param command user command input
+     * @return respective command object based on user input
+     * @throws DukeWrongInputException if system does not understand user input
+     */
+    public static Command parse(String command) throws DukeWrongInputException {
+        // User input
+        String[] commandArr = command.trim().split(" ");
+        command = command.trim();
+        switch(commandArr[0]) {
```

Comment:
Should there be a space between `switch` and `(`?

Reply:
Hello, I don't believe there should be one. Do correct me if I am wrong!

Reply:
You are right, I shall go change it.